.github/workflows/tests/test_raster.py

```diff
@@ -8,17 +8,9 @@
 def test_raster_api():
     """test api."""
     # health
-    resp = httpx.get(
-        f"{raster_endpoint}/healthz", headers={"Accept-Encoding": "br, gzip"}
-    )
-    assert resp.status_code == 200
-    assert resp.headers["content-type"] == "application/json"
-    assert resp.headers["content-encoding"] == "gzip"
```

Comment:
I'm not sure how it worked before, but by default we are not compressing response smaller than 500bytes